javascript/code-challenges/multiBracketValidation/README.md

```diff
@@ -0,0 +1,17 @@
+# Challenge Summary
+Multi bracket validation
+
+## Challenge Description
+Accept a string with brackets and return true if the string contains balanced open and close bracket types, else return false
+
+## Approach & Efficiency
+change string to array, iterate through array.
+push open brackets into a stack, when closing bracket found, compare to top of stack.
+
+O(n) time & space
+
+## Solution
+
+- [Multi Bracket Validation](multi-bracket-validation.js) 
+
+![Whiteboard](assets/codechallenge13whiteboard.png)
```

Comment:
Totally okay to time box yourself and take your time with it! When you have some time to come back to this, get code and some tests written, and I will regrade for you!